src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,94 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int VALID_AGE = 18;
+    private static final String VALID_PASSWORD = "password";
+    private static final String VALID_LOGIN = "admin admin";
+    private RegistrationService registrationService = new RegistrationServiceImpl();
```

Comment:
[Figure out which way is better to initialize your RegistrationServiceImpl class instance: @BeforeEach or @BeforeAll, what’s the difference?](https://mate-academy.github.io/jv-program-common-mistakes/java-core/junit/registration-validation#common-mistakes-jv-registration-validation)